src/app/main/component/ubs/components/ubs-order-details/ubs-order-details.component.ts

```diff
@@ -18,6 +18,7 @@ import { MatDialog } from '@angular/material/dialog';
   styleUrls: ['./ubs-order-details.component.scss']
 })
 export class UBSOrderDetailsComponent extends FormBaseComponent implements OnInit, OnDestroy {
+  @ViewChild('activateBtn') activateBtn: ElementRef;
```

Comment:
instead of getting DOM el with ViewChild for only 1 disabled prop, you can create varibale and use it like
[disabled]="!certificate.valid || certificate.pristine || showTotal < 500 || !certificate.value.length && MyVarHere"

Reply:
done